devito/ir/clusters/analysis.py

```diff
@@ -135,7 +135,7 @@ def _callback(self, clusters, d, prefix):
         # so that `i`'s trip count can safely be rounded up
         # Note: autopadding guarantees that the padding size along the
         # Fastest Varying Dimension is a multiple of the SIMD vector length
-        functions = [f for f in scope.functions if f.is_Tensor]
+        functions = [f for f in scope.functions if f.is_AbstractFunction]
```

Comment:
So why this relaxation?

Reply:
oh no, actually just `is_Tensor` being completely redundant, ie hitting same cases as `is_AbstractFunction` (which was more widespread and "sensible" as name)